lib/ansible/plugins/lookup/conjur_variable.py

```diff
@@ -0,0 +1,155 @@
+# (c) 2018, Jason Vanderhoof <jason.vanderhoof@cyberark.com>, Oren Ben Meir <oren.benmeir@cyberark.com>
+# (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: conjur_variable
+    version_added: "2.5"
+    short_description: Fetch credentials from CyberArk Conjur.
+    description:
```

Comment:
Can we update the `description` text to be a little more explicit that it is the controlling host (the machine running Ansible) and not the target node that must have an established Conjur identity for the lookup plugin to work?

Right now it's a bit confusing because the `requirements` is clear about that but the description just says `...the server's Conjur identity` without saying what 'the server' is in this context.

Reply:
done